### ISSUE TITLE
Bug: Auto-assign of No-show tasks is going to POAs (we think)

### ISSUE THREAD

**Issue by MeredithStewart** [department-of-veterans-affairs/caseflow#10344]
No show admin tasks look like they might be assigned to POAs instead of Hearing Coordinators. 

Shelly to reach out to Hearing Branch to confirm that Hearing coordinators that were listed on the docket are getting tasks.

https://appeals.cf.ds.va.gov/organizations/hearing-admin

Screenshot from the tab "assigned" -- none of the no-show tasks are showing assigned to individual Hearing Coordinators. 

<img width="248" alt="Screen Shot 2019-04-05 at 12 54 38 PM" src="https://user-images.githubusercontent.com/36775274/55643801-08795a80-57a2-11e9-9c07-da754f02f9df.png">

**Comment by MeredithStewart** [comment]
Add screenshot of search with Assigned to POA

**Comment by sharonwarner** [comment]
I confirmed that all NoShowHearingTasks are assigned to the hearing admin organization.

```
NoShowHearingTask.all.map { |task| task.assigned_to }.uniq
[#<HearingAdmin id: 225, type: "HearingAdmin", name: "Hearing Admin", role: nil, url: "hearing-admin", participant_id: nil>]
```

We want to update our assignment logic for the NoShowHearingTask such that if the hearing's hearing day has a coordinator, we assign the NoShowHearingTask to them, if not, we assign it to the hearing admin organization. I think this is lower priority than some other work, so I'm unassigning this ticket for now.

**Comment by MeredithStewart** [comment]
How do we get the front end fixed to show that's it's assigned to Hearing Admin? We've got a ton of no show tasks that showing POA in Assigned to on the No Show tasks.

**Comment by MeredithStewart** [comment]
It also doesn't look like the no-show tasks can be worked. There's a task and a very long hold, but no actions to take on the task. 

For this one: https://appeals.cf.ds.va.gov/queue/appeals/3164302 

<img width="1041" alt="Screen Shot 2019-04-08 at 3 22 05 PM" src="https://user-images.githubusercontent.com/36775274/55750819-2a791400-5a12-11e9-8bb7-98af1ec674d9.png">

**Comment by sharonwarner** [comment]
Sounds like there are three different bugs here:

1) No Show Hearing Tasks are showing up as assigned to the wrong organization. Lowell is working on a fix for this.
2) Hearing admins can't work No Show Hearing Tasks while they're on hold. I've submitted a PR to fix this (#10367).
3) The on hold time for No Show Hearing Tasks is too long. Nobody is working on this yet.

**Comment by MeredithStewart** [comment]
Tested fix for no show admin actions to have actions -- they all now have actions that users can take! Yay!
